tox.ini

```diff
@@ -44,12 +44,13 @@ commands =
     ./validate.sh secret_store
     ./validate.sh invoke-simple
     ./validate.sh invoke-custom-data
-    ./validate.sh demo_actor
+    ; ./validate.sh demo_actor
+    ./validate.sh demo_workflow
     ./validate.sh invoke-binding
     ./validate.sh grpc_proxying
     ./validate.sh w3c-tracing
     ./validate.sh distributed_lock
-    ./validate.sh configuration
+    ; ./validate.sh configuration
```

Comment:
Any reason to leave these commented out/

Reply:
Yeah, these tests are failing with 1.10.x Dapr version. These should be fixed with 1.11.x version though, So, will remove these commented outs, once 1.11.x can be pinned and validation can run against them.